### ISSUE TITLE
GitHub release task can't get initial commit

### ISSUE THREAD

**Issue by stijnmoreels** [arcus-azure/azure-devops-templates#19]
**Describe the bug**
No correct release can be made for the Arcus webapi repo because the GitHub release task fails with an `can't find initial release` error.
All the required parameters are correctly filled out (repo name and release notes).

**To Reproduce**
Run release of Arcus web app.

**Expected behavior**
A new GitHub release is created.

**Comment by tomkerkhove** [comment]
I would take this one as prio imo

**Comment by stijnmoreels** [comment]
> I would take this one as prio imo

Yeah, but have actually no idea from where this comes from. Have done some research first before I added this issue.
I thought that maybe you would recognize this or some sort and would therefore know what the problem was.

**Comment by stijnmoreels** [comment]
Hmm, I think it has something to do with the repo itself. Since the security repo succeeded with the same YAML template for GitHub releases.
Can you check any permissions, @tomkerkhove ?
